a11y/FrontendAccessibilitySpec.scala

```diff
@@ -0,0 +1,38 @@
+import config.FrontendAppConfig
+import org.scalacheck.Arbitrary
+import play.api.data.Forms._
+import play.api.data.Form
+import play.twirl.api.Html
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryList
+import uk.gov.hmrc.scalatestaccessibilitylinter.views.AutomaticAccessibilitySpec
+import views.html._
+import views.html.templates.Layout
+import viewmodels.govuk.summarylist._
+import views.html.auth.SignedOutView
+class FrontendAccessibilitySpec extends AutomaticAccessibilitySpec {
+
+  val viewPackageName                                   = "views"
+  val layoutClasses                                     = Seq(classOf[Layout])
+  val appConfig: FrontendAppConfig                      = app.injector.instanceOf[FrontendAppConfig]
+  //implicit val arbConfig: Arbitrary[FrontendAppConfig] = fixed(appConfig)
+  val list: SummaryList                                 = SummaryListViewModel(rows = Seq.empty)
+  implicit val arbSummaryList: Arbitrary[SummaryList]   = fixed(list)
+  private val booleanForm: Form[Boolean]                = Form("value" -> boolean)
+  implicit val arbForm: Arbitrary[Form[_]]              = fixed(booleanForm)
```

Comment:
this is an arbitrary for form of any type but it uses the boolean form, how does this work/do we need this?

Reply:
It is used for both the boolean and the arbitrary form there are two that need it.

Reply:
ukims and nirms